core/src/main/java/org/apache/calcite/sql/fun/SqlLibraryOperators.java

```diff
@@ -1610,6 +1610,36 @@ private static RelDataType deriveTypeMapFromEntries(SqlOperatorBinding opBinding
           OperandTypes.STRING_STRING,
           SqlFunctionCategory.TIMEDATE);
 
+  /** The "TO_TIMESTAMP_LTZ" function returns a Calcite
```

Comment:
It's worth spelling out that a `TIMESTAMP WITH LOCAL TIME ZONE` is stored as millis since UTC epoch.

Describe which time zone is used for conversion from `DATE` and `TIMESTAMP`.

Reply:
Done